packages/tasks/src/model-libraries.ts

```diff
@@ -163,6 +163,9 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://fasttext.cc/",
 		snippets: snippets.fasttext,
 		filter: true,
+		countDownloads: {
+			term: { path: "model.bin" },
```

Comment:
Some repos don't use `model.bin`, but other names

* `model_v1.bin` https://huggingface.co/cis-lmu/glotlid/tree/main 
* `debate2vec.bin` https://huggingface.co/Hellisotherpeople/debate2vec 
* and [so on](https://huggingface.co/NbAiLab/nb-nordic-lid/blob/main/nb-nordic-lid.bin))

Maybe better to use a wildcard here, WDYT?

Reply:
> Some repos don't use `model.bin`, but other names
> 
> * `model_v1.bin` https://huggingface.co/cis-lmu/glotlid/tree/main
> * `debate2vec.bin` https://huggingface.co/Hellisotherpeople/debate2vec
> * and [so on](https://huggingface.co/NbAiLab/nb-nordic-lid/blob/main/nb-nordic-lid.bin))
> 
> Maybe better to use a wildcard here, WDYT?

Thanks for your comment @osanseviero 

Collating all the comments:
- Some popular fastText models are not using model.bin as @osanseviero suggested
- the naming convention fastText documentation is not deployed (and is not within our control) so users are not aware of it
- fastText model (while there is still a lot) is not a majority in the hub, so the ES query efficiency (due to wildcard vs exact) might not be a major concern to ES

Perhaps, we could go for the wildcard query for now? @Wauplin @julien-c wdyt?

Reply:
Let's go with the wildcard then, thanks for the input @osanseviero @kenhktsui!

Reply:
Thanks @Wauplin. I have pushed the commit.

Reply:
personally i think it's good to enforce some level of filename convention. That's what we do for other libraries too.

But up to you

Reply:
i.e., why even enforce `.bin` file extensions then? We could set `*` as wildcard then?

Reply:
Sorry for the conflicting guidelines @kenhktsui :smile: 

Let's merge it as is with the `*.bin` wildcard.